docs/usage/shoot_operations.md

```diff
@@ -0,0 +1,40 @@
+# Trigger shoot operations
```

Comment:
Nice docs.
I will remove the https://github.com/gardener/documentation/blob/master/website/documentation/050-tutorials/content/howto/maintain-shoot/_index.md#shoot-annotations and link this file as separate page.
Is it okay for you?

Reply:
Yes, sure, thanks! Didn't know that this already exists.